packages/flame_bloc/pubspec.yaml

```diff
@@ -2,6 +2,7 @@ name: flame_bloc
 description: Bloc integration to Flame games
 version: 1.1.0
 homepage: https://github.com/flame-engine/flame/tree/main/packages/flame_bloc
+publish_to: none
```

Comment:
I don't think these changes are correct

cc @spydon

Reply:
I just tried undoing these changes in [e49616](https://github.com/flame-engine/flame/pull/1337/commits/e49616130bafbf384dd73bb9d4e9a5bc4ff256e9), but it immediately led to [analyzer failure](https://github.com/flame-engine/flame/runs/5080118337?check_suite_focus=true) in the CI. So I believe these will have to stay until #1354 is working correctly.

Reply:
I would rather wait for that PR to be merged than to merge this changes as it is, because then we will have to go through the places and revert

Reply:
I guess it would depend on how close #1354 is to completion, so probably @spydon knows best. I notice that that PR already fixes some of the `pubspec.yaml` files, so it's probably not a big deal to fix few more within that same PR.